src/classes/EMPReplicantProfile.cls

```diff
@@ -0,0 +1,104 @@
+public class EMPReplicantProfile {
+
+	public ApexPages.StandardController stdController;
+    public EMP_Organization_Member__c organizationMember;
+    public EMP_Farm_Diagnostic_and_Targets__c[] farmTargetsDiagnostics = new List<EMP_Farm_Diagnostic_and_Targets__c>();
+    public EMP_Farm_Follow_Up__c[] farmFollowUps = new List<EMP_Farm_Follow_Up__c>();
+    public EMP_Technical_Assistance__c[] farmTechnicalAssistance = new List<EMP_Technical_Assistance__c>();
+    public EMP_Farm_Baseline__c[] farmBaseline = new List<EMP_Farm_Baseline__c>();
+    public EMP_Organization_Member__c organizationMemberId;
+    public List<EMP_Farm_BPA_Checklist__c> bpaCheckList;
+
+	public EMPReplicantProfile(ApexPages.StandardController controller) {
+		this.stdController = controller;
+        this.organizationMember= (EMP_Organization_Member__c)controller.getRecord();  
+        EMP_Organization_Member__c [] organizationMemberIds = [Select Id From EMP_Organization_Member__c Where Id =:organizationMember.Id];
+        if (organizationMemberIds.size() > 0) {
+	        organizationMemberId = organizationMemberIds[0];
+	    }
+    }
+
+    public EMP_Organization_Member__c getOMId() {
+	    return organizationMemberId;
+	}
+
+    public List<EMP_Farm_Diagnostic_and_Targets__c> getTargetDiagnostic() {
+    	if (organizationMember != null) {
+        	EMP_Farm_Diagnostic_and_Targets__c[] farmTargetsDiagnostics = [Select Id, Beans_To_Be_Harvested_Kg__c, BPA_Not_Implemented__c, Commercialization_to_date__c, Commercialized_Kg__c, Commitment_Agri_Input_Storage__c, Commitment_Agrochem_Management__c, Commitment_Registration__c, Commitment_Soil_Water_Analysis__c, Crop_Area__c, Crop_Stage__c, Farmer_signature__c, Green_Dried_Beans__c, kilos_plant_replica__c, kilos_will_plant_replica__c, Number_of_plots__c, Other_Destination_Sold_Product__c, Other_Reason_Agri_Input_Storage__c, Other_Reason_Agrochem_Management__c, Other_Reason_Not_Registration__c, Other_Reason_Soil_Water_Analysis__c, Other_Use_of_Resources__c, plant_spacing_cm__c, Price_Kg__c, Price_per_Kg_to_date__c, Production_Commercialized__c, Production_Kg__c, Production_to_date_Kg__c, Productive_sector__c, Reason_Agri_Input_Storage__c, Reason_Agrochem_Management__c, Reason_Registration__c, Reason_Soil_Water_Analysis__c, Requires_Org_Resources__c, row_spacing_cm__c, Use_of_Resources__c, Visit_Date__c, Visit_Number__c
```

Comment:
For large queries like this, it might be easier to include each column on a separate line.